bin/worker.js

```diff
@@ -17,6 +17,8 @@ export default {
    * @returns
    */
   async fetch(request, env, ctx, { retrieveFile = defaultRetrieveFile } = {}) {
+    // Worker entry timestamps
+    const t0 = performance.now()
```

Comment:
Let's use a more descriptive name, please. E.g. 

```suggestion
    const startedAt = performance.now()
```

Reply:
+1 on more descriptive naming